App/Sources/Feature/InputPostFeature/Source/InputPostView.swift

```diff
@@ -16,27 +16,25 @@ struct InputPostView: View {
 
     var body: some View {
         ScrollView {
-            VStack(spacing: 0) {
-                InputFormView(
-                    epic: "게시글",
-                    state: "추가",
-                    settingButtonAction: {
-                        viewModel.isPresentedDetailSetting(state: true)
-                    },
-                    finalButtonAction: {
-                        viewModel.addPost()
-                        dismiss()
-                    },
-                    title: Binding(
-                        get: { viewModel.postTitle },
-                        set: { title in viewModel.postTitle = title }
-                    ),
-                    text: Binding(
-                        get: { viewModel.postText },
-                        set: { text in viewModel.postText = text }
-                    )
+            InputFormView(
+                epic: "게시글",
+                state: "추가",
+                settingButtonAction: {
+                    viewModel.isPresentedDetailSetting(state: true)
+                },
+                finalButtonAction: {
+                    viewModel.addPost()
+                    dismiss()
+                },
+                title: Binding(
+                    get: { viewModel.postTitle },
+                    set: { title in viewModel.postTitle = title }
+                ),
```

Comment:
```suggestion
                title: $viewModel.postTitle,
```

get이나 set이 단순하게 되어있으면 그냥 $로 바인딩하는거 어떤가요

Reply:
b2ef8fc74c25a19c90c6f12ac8b427a3dbe3ee2e

set부분에 viewModel에있는 updatePostTitle함수를 사용하도록 수정했어요!